Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.module.scss

```diff
@@ -1,28 +1,47 @@
 @use 'src/styles/global';
 @use 'src/styles/components';
 @use 'src/styles/fonts';
+@use 'src/styles/colors';
+
+.backBtn {
+  border: 0;
+  margin-left: -20px;
+  margin-right: -30px;
+}
+
+.btnContainer {
+  display: flex;
+  flex-direction: row;
+}
 
 .code {
   @extend %navigation;
 
   border-left: 0;
-  flex: 7;
 }
 
 .detailsWrapper {
   @extend %wrapper;
   @extend %columnsThree;
 }
 
+.leftArrow {
+  color: colors.$light-blue-color;
+  margin-bottom: -5px;
+}
+
+.itemInvisible {
+  visibility: hidden;
+}
+
 .navigation,
 %navigation {
   align-content: center;
   flex: 3;
 }
 
-.taskHeading {
-  margin-bottom: 24px;
-  text-align: center;
+.retestButton {
+  margin: 0 0 0 24px;

```

Comment:
Is it needed to use 0 values and 24px rather than margin-top (or whatever the 4th value is for)?

Reply:
I am not sure, but the retest button is not in the scope of my issue. It was merged from v2 into my branch.